apps/wizard/app_pages/chart_animation.py

```diff
@@ -62,103 +74,141 @@
+            year_min, year_max = st.select_slider(
+                "Select year range",
+                options=st.session_state.chart_animation_years,
+                value=(query_parameters["year_min"], query_parameters["year_max"]),
+            )
 
         # Get the selected subset of years.
         years = [year for year in st.session_state.chart_animation_years if year_min <= year <= year_max]
 
-        st.session_state.chart_animation_max_num_years = st.number_input(
-            "Maximum number of years",
-            value=MAX_NUM_YEARS,
-            help="Maximum number of years to generate images for (to avoid too many API call).",
-        )
-
     if len(years) > st.session_state.chart_animation_max_num_years:
         st.error(
             f"Number of years in the chart ({len(years)}) is higher than the maximum number of years ({st.session_state.chart_animation_max_num_years}). You can either increase the maximum number of years or select a smaller range."
         )
         st.stop()
 
-    # SHOW OPTIONS FOR IMAGE GENERATION
-    with st.container(border=True):
-        # Tab and year range settings.
-        def add_icons_to_tabs(tab_name):
-            if tab_name == "map":
-                return ":material/map: map"
-            elif tab_name == "chart":
-                return ":material/show_chart: chart"
-            return f":material/{tab_name}: {tab_name}"
-
-        # tab = st.radio("Select tab", ["map", "chart"], horizontal=True)
-        tab = st.segmented_control(
-            "Select tab", ["map", "chart"], format_func=add_icons_to_tabs, default=query_parameters["tab"]
+    # 2.2/ SHOW OPTIONS FOR IMAGE GENERATION
+    with st.expander("**Output settings**", expanded=True):
+        with st_horizontal():
+            # Choose: GIF or Video
+            output_type = st.segmented_control(
+                "Output format",
+                ["GIF", "Video"],
+                default="GIF",
+            )
+            # Social media?
+            output_style = st.segmented_control(
+                "Output style",
+                ["Classic", "Square format"],
+                default="Classic",
+                help="Use 'square format' for mobile or social media.",
+            )
+            social_media_square = output_style == "Square format"
+
+        st.session_state.chart_animation_gif_file = st.session_state.chart_animation_gif_file.with_suffix(
+            ".gif" if output_type == "GIF" else ".mp4"
         )
 
+        # If chart, show option to just show single year
         if tab == "chart":
-            year_range_open = st.toggle(
-                "Year range open",
-                value=query_parameters["year_range_open"],
-                help="Only relevant for the chart view. If checked, the year range will be open. Uncheck if you want to generate a sequence of bar charts.",
+            year_range_open = not st.toggle(
+                "Show only current year",
```

Comment:
"Show only current year" is not exactly the right explanation. For a line chart, it means showing just one year at a time. But for a scatter chart, it means showing bubbles instead of lines. That's way I called it "year range open/close". Maybe you can think of a better name, but "only current year" is a bit misleading.

Reply:
Changing it to "show single year" instead. Improved help message.